types/selenium-webdriver/test/bidi/logInspector.ts

```diff
@@ -0,0 +1,15 @@
+import * as webdriver from "selenium-webdriver";
+import LogInspector from "selenium-webdriver/bidi/logInspector";
+
+function TestLogInspector() {
+    const driver: webdriver.WebDriver = new webdriver.Builder().build();
+    const logInspector = LogInspector(driver);
```

Comment:
```suggestion
import * as webdriver from "selenium-webdriver";

function TestLogInspector() {
    const driver: webdriver.WebDriver = new webdriver.Builder().build();
    const logInspector = webdriver.LogInspector(driver);
```

I found [some docs](https://developer.chrome.com/blog/webdriver-bidi-2023/#logging-powered-by-webdriver-bidi) on this feature's interface, and it looks like `LogInspector` is a top-level import. So I think that you'll need to export it [from the index file](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/39496b0e316dd0a4d9a2e3de8cd6c5b2489b2f22/types/selenium-webdriver/index.d.ts) to match the real API.

Reply:
Oh neat! Yep, adding it there 👍